### PR TITLE
Missing translation

### PR DESCRIPTION
**Pull request by core23**
See https://github.com/sonata-project/SonataPageBundle/blob/master/Resources/views/Page/create.html.twig#L25

### CODE REVIEW

File: Resources/translations/SonataPageBundle.fr.xliff [hunk 1]
```diff
@@ -796,6 +796,10 @@
           <source>page.composer_status_error</source>
           <target>Une erreur est survenue durant la sauvegarde du statut du bloc.</target>
       </trans-unit>    
+      <trans-unit id="198" resname="message_page_does_not_exist">
+          <source>message_page_does_not_exist</source>
+          <target>message_page_does_not_exist</target>
```
Comment (OskarStark) [inline Resources/translations/SonataPageBundle.fr.xliff]: ping @Soullivaneuh can you help? :smile:

Comment (Soullivaneuh) [inline Resources/translations/SonataPageBundle.fr.xliff]: "Le message de page n'existe pas."

But not sure about that. ping @rande

Comment (meyerbaptiste) [inline Resources/translations/SonataPageBundle.fr.xliff]: "Cette page n'existe pas." It's better @Soullivaneuh :wink:

Comment (Soullivaneuh) [inline Resources/translations/SonataPageBundle.fr.xliff]: @meyerbaptiste Yeah, but it's the `message_` that confuse me.

**Comment by core23** [comment]
Added French translation

**Comment by OskarStark** [comment]
thank you @core23

**Pull request merged**